src/runtime/program_loader.zig

```diff
@@ -76,7 +77,7 @@ pub const LoadedProgram = union(enum(u8)) {
 };
 
 pub fn loadIfProgram(
-    allocator: std.mem.Allocator,
+    gpa: std.mem.Allocator,
```

Comment:
Why did you change this to gpa? Is it just to correlate it with the gpa param in loadAndExecuteTransaction?